.github/workflows/deploy_on_change_to_master.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Deploy Master
+      run: git push heroku master
```

Comment:
Is there some setup to make this work? Its complaining because it doesn't know about the `heroku` remote.

Reply:
yea prolly. don't worry bout this one too much. still workin it out